UPDATING.md

```diff
@@ -85,6 +86,26 @@ To see which versions of Python we test the SDK against, please look at our
 The 1.2.0 version introduces a breaking change, which changes the way of interacting with the SDK from synchronous to asynchronous, using [the `asyncio` foundational library](https://docs.python.org/3.7/library/asyncio.html) to provide support for `async`/`await` syntax.
 Because of this breaking change, every call that interacts with the Ably REST API must be refactored to this asynchronous way.
 
+Important Update:
+- If you want to keep using old synchronous style API, import `AblyRestSync` client instead.
+- This is applicable only for Ably REST APIs.
+
+```python
+from ably.sync import AblyRestSync
```

Comment:
This will need change to the top level file `__init__.py` under `ably` package. Currently, `unasync.py` doesn't make changes to the input source files.
Also, I feel making package import explicit as `ably.sync` will make sure, future imports for all `sync` classes come under this package. 
I just don't want `generated code` to interfere with input source code at import level.

Reply:
ok yeah that's a fair point, we can leave it like this for now 👍